OMPython/__init__.py

```diff
@@ -906,29 +916,33 @@ def loadLibrary(self, verbose):
                         libname = "".join(["loadModel(", element[0], ", ", "{", "\"", element[1], "\"", "}", ")"])
                     result = self.sendExpression(libname)
                 else:
-                    print("| info | loadLibrary() failed, Unknown type detected: ", element , " is of type ",  type(element), ", The following patterns are supported\n1)[\"Modelica\"]\n2)[(\"Modelica\",\"3.2.3\"), \"PowerSystems\"]\n")
+                    raise ModelicaSystemError("loadLibrary() failed, Unknown type detected: " +
+                                              "{} is of type {}, ".format(element, type(element)) +
+                                              "The following patterns are supported:\n" +
+                                              "1)[\"Modelica\"]\n" +
+                                              "2)[(\"Modelica\",\"3.2.3\"), \"PowerSystems\"]\n")
                 ## Show notification or warnings to the user when verbose=True OR if some error occurred i.e., not result
-                if verbose or not result:
-                    print(self.requestApi('getErrorString'))
+                if not result:
```

Comment:
Check for `self._verbose` flag here. `loadLibrary` raises some warnings and notifications that are good to know specially when `verbose` is set.